src/core/request.ts

```diff
@@ -290,7 +290,7 @@ export class Request {
     throw new Exceptions.RequestError(json);
   }
 
-  send(options = {}, attempts = 0): Bluebird<any> {
+  send(options = {}, attempts = 0): any {
```

Comment:
Why did you remove `Bluebird<any>` in favor of `any`? I changed it back locally and everything remains fine. Do you have any reasons to keep it `any`?

Reply:
As i see, there is no reason to keep it `any`, so i just changed it back. It is important because we use bluebird [filtered catch](http://bluebirdjs.com/docs/api/catch.html#filtered-catch) feature, that is not provided by standard promises

Reply:
You were right